docs/sources/developers/plugins/migration-guide.md

```diff
@@ -308,6 +319,32 @@ import { cx, css } from 'emotion';
 import { cx, css } from '@emotion/css';
 ```
 
+### Update needed for app plugins using dashboards
+
+To make side navigation work properly - app plugins targeting Grafana 8.+ and using navigation bar need to adjust their `plugin.json` and all dashboard json files to have a matching `uid`.
```

Comment:
Don't have time now but maybe mention app including dashboard with a link to it in the navigation bar using `addToNav` property.

Reply:
@marefr added the addToNav part and link to it's definition